### PR TITLE
Base model creation

### LINKED ISSUE #5: Create model

### PR DESCRIPTION
**Pull request by J-Canedo**
Created a base model based on CNN classes exercises that recognizes images from cifar10 dummy dataset from tensorflow.

model is running with ~55% accuracy but not much time was lost on fine-tunning since we are going to plug our database on it and then refine

Code was transcrypted to VS Code but Jupyter notebook also contains learning curves visualizations for overfitting assessment and confusion matrix to understand each classes results

Next steps are to plug our datasets in the model to test and fine-tune

### CODE REVIEW

File: LibrAI Model Mockup Notebook.ipynb
Comment (cajudeleite) [inline LibrAI Model Mockup Notebook.ipynb:1]: Rename it to librAI

File: .ipynb_checkpoints/LibrAI Model Mockup Notebook-checkpoint.ipynb
Comment (cajudeleite) [inline .ipynb_checkpoints/LibrAI Model Mockup Notebook-checkpoint.ipynb:1]: Same here

**Review by cajudeleite — CHANGES_REQUESTED** [review]

**Review by cajudeleite — APPROVED** [review]
Good job
